frontend/demo/src/actions/conversations/index.ts

```diff
@@ -1,9 +1,12 @@
 import {Dispatch} from 'redux';
 import {createAction} from 'typesafe-actions';
-import {doFetchFromBackend} from '../../api/airyConfig';
-
-import {Conversation, ConversationPayload, conversationsMapper} from '../../model/Conversation';
-import {ResponseMetadata} from '../../model/ResponseMetadata';
+import {
+  AiryHttpClient,
+  Conversation,
+  conversationsMapper,
```

Comment:
Gut feel tells me the mapper is a concern of the demo app more than the library. But, as I said, it's not structured feedback as I'm not sure I understand the code well enough so the comment is more to initiate a conversation around it